Sources/TabView.swift

```diff
@@ -347,6 +347,15 @@ extension TabView {
             underlineView = UIView(frame: CGRect(x: itemView.frame.origin.x + options.underlineView.margin, y: itemView.frame.height, width: itemView.frame.width - options.underlineView.margin * 2, height: options.underlineView.height))
             underlineView.backgroundColor = options.underlineView.backgroundColor
             containerView.addSubview(underlineView)
+        case .circle:
```

Comment:
Rename  `func setupUnderlineView()` as well

Reply:
and Rename `func updateUnderlineViewPosition()`, `func resetUnderlineViewPosition()`, `func moveUnderlineView()`, too.